src/components/AddBook.jsx

```diff
@@ -1,41 +1,53 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/books/booksSlice';
+import { nanoid } from 'nanoid';
+import { addBook, getBooks } from '../redux/books/booksSlice';
 
 const AddBook = () => {
-  let title = '';
-  let author = '';
-  const item_id = 'temporary';
+  //  const { books } = useSelector((store) => store.books);
+  const [title, setTitle] = useState('');
```

Comment:
- [OPTIONAL] KIndly make sure to remove all comments in your codebase in order to keep it clean and professional.